plugins/proof-pr/skills/proof-pr-workflow/attempt-fix.py

```
@@ -0,0 +1,513 @@
+        }
+
+
+class AutoFixer:
+    """Automatically fix common compilation errors"""
+
+    # Common type rename patterns from OpenShift API changes
+    TYPE_RENAME_PATTERNS = [
+        # Sigstore verification types (from api#2626 simulation)
+        (r'\bPolicy\b', 'ImageSigstoreVerificationPolicy'),
+        (r'\bPKI\b', 'ImagePolicyPKIRootOfTrust'),
+    ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)